packages/strapi-plugin-content-type-builder/admin/src/components/TableListRow/index.js

```diff
@@ -48,7 +71,13 @@ class TableListRow extends React.Component {
 
   toggleModalWarning = () => this.setState({ showWarning: !this.state.showWarning });
 
-  handleShowModalWarning = () => this.setState({ showWarning: !this.state.showWarning });
+  handleShowModalWarning = () => {
+    if (this.props.canOpenModalAddContentType || this.props.rowItem.isTemporary === true) {
+      this.setState({ showWarning: !this.state.showWarning });
+    } else {
+      strapi.notification.info(`${pluginId}.notification.info.contentType.creating.notSaved`);
```

Comment:
Don't you want to use the pluginId variable in the whole file ?
ex: `/plugins/content-type-builder/models/...` paths